package.json

```diff
@@ -41,15 +41,15 @@
     "@types/node": "^9.4.6",
     "@types/node-fetch": "^2.3.4",
     "@types/openfin": "^43.0.1",
-    "@types/puppeteer": "1.3.0",
+    "@types/puppeteer": "^1.19.1",
```

Comment:
This is ahead of the puppeteer version because there was no types package since 1.6.4. I thought it was best to have the stuff for .17, and the .18 and .19 changes weren't relevant to any of the stuff we use anyway.